onnxruntime/python/tools/gen_doc.py

```diff
@@ -114,10 +114,11 @@ def display_schema(schema, versions):  # type: (OpSchema, Sequence[OpSchema]) ->
     s = ''
 
     # doc
-    if schema.doc:
```

Comment:
Can we move this file to tools/python? The top-level python dir is meant for the python bindings and this script is unrelated to the bindings.

Reply:
done